### DISCUSSION TITLE
Downloaded files not there

### DISCUSSION THREAD

**Discussion by bakereliz** [aclap-dev/video-downloadhelper#1166] [Help and Q&A] [answered]
### Your question

Hi. I am using a beta version 9.1 or something like that with the companion app on Windows 11 Surface Book 3 and Firefox. I have used VDH for many years now (but this is the first time I have used it in a few months). For some reason, when I download a video, it does not show up in File Explorer. If I try to download it again, it does show in Save File As in the VDH file listing as having been downloaded into the correct folder. Where can my downloads be? Are they really there at all?

Thanks for your help, Elizabeth 

### Addon & CoApp version and details

version: 9.1.0.41
channel: beta
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): b890c67e6ec54c85
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\baker",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
hi,
isn't it in your default download directory? : https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#change-the-default-download-directory
jerome

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

That's awfully strange.  I've never had that problem, & we're running just about the same setup: Windows 11 Pro 64-bit version 23H2 build 22631.4037, Firefox 129.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

Maybe your file manager settings are hiding files.  When a download completes, a green entry appears in the VDH menu.  When you click the small control on the right side of the green button, it's supposed to open a new file manager window on the directory containing the downloaded file.  Is that not what happens for you?  You might post a sequence of screenshots showing your VDH download & the empty directory in the file manager.  Maybe somebody will notice something that would help you.  Here's an example of what I'm asking for:

My example URL is https://www.youtube.com/watch?v=-4SKjqg1MKM.

![#01](https://github.com/user-attachments/assets/7009f384-22ad-4d71-9210-2bb8fb9548c3)
![#02](https://github.com/user-attachments/assets/7ace28c5-1df9-4a9c-8a3c-6d3ef950e55b)
![#03](https://github.com/user-attachments/assets/ca0ca240-4d67-4b40-b012-e95e60677808)
![#04](https://github.com/user-attachments/assets/ad56a170-118f-47b6-ba33-589831e51dfb)
![#05](https://github.com/user-attachments/assets/537ca77a-7d05-4439-84e2-5d7b3f555259)

Actually, I cheated a bit.  I already had the file manager open on the download directory.  I think there are some shortcomings in the Windows APIs for the file manager.  You can see in that last image that my file manager window has several open tabs.  Truth is that if I had positioned my file manager on one of the other tabs, that's what VDH would have taken me to when I clicked the control in the green button.  I'm not so sure this is entirely a bug in VDH.  In any case, as you can see, I downloaded a file & I can see it in the file manager.

Your turn.

**Comment by bakereliz** [comment]
Thanks, Jerome. My default download directory is set correctly, but, wow! My UI looks NOTHING like the info you sent me!--elizabeth

I don't know what the heck I'm doing, but I'm sure having a grand time doing it! 

    On Monday, August 19, 2024 at 12:36:08 AM PDT, jcfrog ***@***.***> wrote:  
 
 


hi,
isn't it in your default download directory? : https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#change-the-default-download-directory
jerome

**Comment by bakereliz** [comment, marked as answer]
Thanks for responding. I always have my file manager settings set to show hidden files. Have for years.My interfact looks NOTHING like yours. OR the other poster who responded to me. 
With my version, I click on VDH, get a listing of available resolutions, click the arrow to the right of the one I want, choose Download from a dropdown box to save to location, give it a name and click save. My interface hasn't changed since I originally installed it all those years ago. 
YOUR interface looks MUCHmore updated.
I'd give you some screenshots of MY UI: 

After I selected one of these resolutions I use the Download option to download to my hard drive and set a destination.


I save the file in the proper place and give it a name. Here below you can see a number of files I downloaded which do NOT show up in File Explorer.


Here is what VDH looks like while downloading.



When it's finished downloading, the icon looks the same but without the number indicator. If I click on it, I get a list of available resolutions, not an option to open the downloaded file.

Maybe I should just uninstall and reinstall fresh.
--elizabeth


I don't know what the heck I'm doing, but I'm sure having a grand time doing it! 

    On Monday, August 19, 2024 at 12:50:49 AM PDT, GrampaWildWilly ***@***.***> wrote:  
 
 


I do not work for VDH. I'm just another user.

That's awfully strange. I've never had that problem, & we're running just about the same setup: Windows 11 Pro 64-bit version 23H2 build 22631.4037, Firefox 129.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

Maybe your file manager settings are hiding files. When a download completes, a green entry appears in the VDH menu. When you click the small control on the right side of the green button, it's supposed to open a new file manager window on the directory containing the downloaded file. Is that not what happens for you? You might post a sequence of screenshots showing your VDH download & the empty directory in the file manager. Maybe somebody will notice something that would help you. Here's an example of what I'm asking for:

My example URL is https://www.youtube.com/watch?v=-4SKjqg1MKM.

01.png (view on web)
02.png (view on web)
03.png (view on web)
04.png (view on web)
05.png (view on web)

Actually, I cheated a bit. I already had the file manage open on the download directory. I think there are some shortcomings in the Windows APIs for the file manager. You can see in that last image that my file manager window has several open tabs. Truth is that if I had positioned my file manager on one of the other tabs, that's what VDH would have taken me to when I clicked the control in the green button. I'm not so sure this is entirely a bug in VDH. In any case, as you can see, I downloaded a file & I can see it in the file manager.

Your turn.

**Reply by GrampaWildWilly** [reply]
You are using E-mail to interact with this discussion.  Don't.  Sign on to GitHub & update the discussion using your web browser.  That is the only way to get attachments to appear in a discussion.  It looks like you tried to attach some images but because you did it via E-mail, it didn't work.  Your images did not get posted.  And I'd very much like to see them.  So get in here in your browser & try again & let's take it from there.

**Reply by GrampaWildWilly** [reply]
By the way, if you do E-mail posts here, you should delete all quoted material from your E-mail before you send it.  When you fail to delete all that junk, it ends up in the discussion as duplicated material.  That's just wasteful.  These discussions are very much oriented towards viewing them in the web browser.  When you do that, you will see that quoting other posts is really quite wasteful.  I strongly encourage you to quit posting via E-mail.  It's one thing to get E-mails of other posts & look at those in your E-mail.  But when you want to post, it's just better to do it in the web browser.

**Reply by GrampaWildWilly** [reply]
I have a hunch.  Go into your VDH settings and . . .

![#01](https://github.com/user-attachments/assets/b9172308-2020-4f50-979c-d9dfc10bc508)

**Reply by GrampaWildWilly** [reply]
Here's another hunch.  Read this: #814.

**Reply by bakereliz** [reply]
Thanks for the tips/advice. The screenshots I sent are now not really valid anyway, since I have clicked on Enable New UI. My interface looks much more like yours now. However, my VDH is missing the ribbon at the bottom with the settings icon, trash,  etc. Any ideas why I don't have those icons showing?

I WAS able to download a file and then check to see its download location, which was in a totally wacked out place. I moved it to its correct location. When I downloaded the next one, I found the Download As and changed the download location, so it is now saving the file where I wanted it. 

The only thing now is how to get the settings icon to show up!

Thanks so much for all your help.

**Reply by GrampaWildWilly** [reply]
I find it quite odd that you installed VDH 9 as a straight update & did not yourself choose to use the legacy UI.  It is clear that you were using the legacy UI but you apparently didn't choose to do so.  I haven't seen that.  When I installed VDH 9 (the earliest release number) on top of VDH 8 (the last release number), it immediately showed me the new UI.  So the fact that you had the legacy UI is quite baffling.

The settings gear missing is another baffler.  Perhaps going into the Firefox settings (Ctrl+Shift+a) & disabling then re-enabling VDH will make it appear.  I'm guessing now so this is a low probability solution.  Another low probability solution might be to just go into the Wiki where you originally found the beta & install it again.  Once upon a time with VDH 7 I was having a weird problem with things being cut off at the bottom of the VDH menu.  Fiddling with the size of the browser window made things appear properly.  But that was with a release 2 versions back so that would be another low probability thing to try.  As a last resort, I would do a remove on VDH on the Firefox Extensions page, close all your browser windows, launch Firefox again, then install the beta from the Wiki.  I'm just full of low probability ideas tonight.

The wacked out target of your download was probably the initial default download directory, which is somewhere in the directory trees created by a standard Windows install.  It's somewhere under either %USERPROFILE% or "Documents and Settings" in a leaf directory named dwhelper (all lower case).  Once you do one `Download As...`, you regain control over where your downloads are going.  Still, your original problem of directing downloads to a directory where they don't appear is, to use your so appropriate terminology, wacked out.  I talk extensively about the behavior of download directories in that discussion #814 I mentioned earlier.  You would do well to read that.  Things are not quite 100% like they used to be in VDH versions prior to 9.

**Reply by bakereliz** [reply]
Thanks so much for all your help. All is well now. The new UI is working well. It was opening in a sidebar and I finally found the settings icon etc at the very bottom of the page in the sidebar. I did not realize the sidebar extended to the bottom of the page so did not see it at first. I've set VDH to display as a pop-up instead of the sidebar, and all icons are displaying in a shortened form. 

Very happy here. Thanks so much for your help and guidance. I would really like to meet you someday. From your writing, I suspect you are a wonderful, patient, kind person.

--elizabeth

**Reply by GrampaWildWilly** [reply]
`blushing`

I'm also great at conning people into believing good things about me.

I'm glad everything is good now.

**Comment by bakereliz** [comment]
AND you have a sense of humor!!!

I don't know what the heck I'm doing, but I'm sure having a grand time doing it! 

    On Tuesday, August 20, 2024 at 04:04:54 PM PDT, GrampaWildWilly ***@***.***> wrote:  
 
 


blushing

I'm also great at conning people into believing good things about me.

I'm glad everything is good now.
